index.js

```diff
@@ -1,19 +1,13 @@
 const chalk = require('chalk')
 
 if (!process.env.CALIBRE_API_TOKEN) {
-  console.log(
-    chalk.grey(
-      'Please set CALIBRE_API_TOKEN as an environment variable. Create your token via the team API tokens page, or see calibreapp.com/docs/api/tokens for help.'
-    ),
-    chalk.blue(`${process.env.CALIBRE_HOST}/home`)
-  )
-  process.exit()
+  throw new Error('Please set CALIBRE_API_TOKEN as an environment variable. Create your token via the team API tokens page, or see calibreapp.com/docs/api/tokens for help.');
```

Comment:
The `process.exit` will cause failures to not be reported in jest, this also seems more like an error to me honestly and should fail hard.

Reply:
Will it cleanly exit?

Reply:
It will exit with an error code, yes.